HCK04_2018_Seattle/Projects/README.md

```diff
@@ -1,8 +1,20 @@
 # How to create a new project
 
-- Join the weekly [preparation call](../README.md#preparatory-videoconferences) to discuss your idea
-- Post questions about the project idea and team on the [Hackathon forum][forum].
-- When you are ready, add a new entry in the list of **Projects** by creating a new `README.md` file in subfolder in `Projects` folder, and copying contents of [project description template][project-description-template] file into it. Step-by-step instructions for this are:
+## First steps:
+
+- Join the weekly [preparation call](../README.md#preparatory-videoconferences) to discuss your idea.
+- Post questions about project ideas and teams on the [Hackathon forum][forum].
+
+## Designing your team and project:
+
+To facilitate the discussions around project ideas, you can ask the [Hackathon forum maintainers][maintainers]  to create a sub-team for you.  The subteams that already exist are listed [here][subteams].
+- Use the main [Hackathon forum][forum] to discuss project scope and team membership.
```

Comment:
I updated my initial comment with `Use your sub-team forum ...` which would be more coherent considering you suggest to create such a sub-team forum above